CONTRIBUTING.md

```diff
@@ -129,12 +129,12 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 ### Workspace Loader
 
-Workspace loader source code is located in [<this.repo>/workspace-loader](./workspace-loader/) folder.
+Workspace loader source code is located in [https://github.com/eclipse/che-workspace-loader](https://github.com/eclipse/che-workspace-loader) folder.
 Here is the developer workflow if you want to contribute to it:
 
 #### Devfile for workspace loader development
```

Comment:
Maybe this part has to move the a dedicated CONTRIBUTING.md in the workspace loader repo. WDYT ?

Reply:
separate issue? CC @sleshchenko

Reply:
https://github.com/eclipse/che/issues/16432